testing-overhead/k6/basic.js

```diff
@@ -0,0 +1,70 @@
+import http from "k6/http";
```

Comment:
perhaps it would be better to place test specific scripts to a common subdirectory so it would be easier to add more tests

Reply:
that's the entire purpose of the `k6` subdir.